templates/oauth_token_request_denied.html

```diff
@@ -1,20 +1,20 @@
 {%  extends 'base_no_intercom.html' %}
+{% load i18n_icu %}
 
 {% block title %}
-    OAuth Error
+   {% trans_html "py.templates.oauth_token_request_denied.error.Title" default="OAuth Error" %} 
 {% endblock %}
 
 {% block body %}
   <div class="oauth-error">
     <p>
-      The authorization server refused to let you log in.
+      {% trans_html "py.templates.oauth_token_request_denied.serverRefusedtoLogin" default="The authorization server refused to let you log in." %}
```

Comment:
Can we please change _every_ message ID in this file:

* `title`
* `description`
* `suggestRetry`
* `contactUs`